src/app/ir/model/embedding.ts

```diff
@@ -31,6 +31,6 @@ export default class Embedding {
   }
 
   public initCode(): string {
-    return `Embedding(NumEmbeddings=${this.NumEmbeddings}, EmbeddingDim=${this.EmbeddingDim}, PaddingIdx= ${this.PaddingIdx}, MaxNorm= ${this.MaxNorm}, NormType= ${this.NormType}, ScaleGradByFreq= ${this.ScaleGradByFreq}, Sparse= ${this.Sparse}, Weight= ${this.Weight})`;
+    return `Embedding(NumEmbeddings=${this.NumEmbeddings}, EmbeddingDim=${this.EmbeddingDim}, PaddingIdx= ${this.PaddingIdx}, MaxNorm= ${this.MaxNorm}, NormType= ${this.NormType}, ScaleGradByFreq= ${this.ScaleGradByFreq}, Sparse= ${this.Sparse}, weight= ${this.Weight})`;
```

Comment:
looking at the doc, the args should be snake case https://pytorch.org/docs/stable/generated/torch.nn.Embedding.html

also extra space after = (sorry for nitpicking :c)

Reply:
this seems to be true for quite a few other files as far as i can tell

Reply:
I changed all of them just now. Was fixing only loss and optimizers yesterday.